### PR TITLE
 feature request: Implementation of a method for creating a configuration interface.

### PR DESCRIPTION
**Pull request by Ordissimo**
Addition of a method listing the providers, a method listing the options of a provider and method of manual configuration.
I want to implement a configuration interface.

### CODE REVIEW

File: cmd/manualconfig/manualconfig.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	cmd.Root.AddCommand(commandDefintion)
+}
+
+var commandDefintion = &cobra.Command{
+	Use:   "manualconfig",
+	Short: `List all options of provider.`,
```
[the remark this answers is not in the document]
Comment (Ordissimo) [reply cmd/manualconfig/manualconfig.go]: Yes indeed.
Short: `Create new remote with name, type and json:options.`

[the remark this answers is not in the document]
Comment (Ordissimo) [reply cmd/manualconfig/manualconfig.go]: This command is not perfect. For example with Dropbox, I am blocked in console. Ave Drive works fine.

**Comment by Ordissimo** [comment]
Hi,
Yes, that's it.
I do that in the evening.

**Comment by Ordissimo** [comment]
I made some corrections.

**Comment by ncw** [comment]
I've merged this in 0575623dffd60a064be60227ea27851b3cd3113d after squashing and rebasing.

In edfab09eb9c39909cdfea4777698088cfb374f75 I have completed the suite of auto config commands for the rclone config file (and renamed a few things).  Hopefully this will now be sufficient for your needs.

Here are the new commands for config
    
     * create    Create a new remote with name, type and options.
     * delete    Delete an existing remote <name>.
     * dump      Dump the config file as JSON.
     * edit      Enter an interactive configuration session.
     * file      Show path of configuration file in use.
     * providers List in JSON format all the providers and options.
     * show      Print (decrypted) config file, or the config for a single remote.
     * update    Update options in an existing remote.

And here are the changes I made

     * listproviders was renamed to providers
     * listoptions was removed in favour of providing the output in providers
     * jsonconfig was renamed to create
     * an optional parameter was added to the show command

The JSON output has changed slightly, and `rclone config create` no longer takes JSON which I think will be an improvement for other rclone users and just as easy to use for you.

Let me know how you get on

**Comment by Ordissimo** [comment]
Awesome, thank you for the development of this feature.
I will implement the configuration interface ...
I will keep you informed.

**Pull request closed without merging**